### DISCUSSION TITLE
Difference in yhat with and without quantiles specified

### DISCUSSION THREAD

**Discussion by tuttoaposto** [ourownstory/neural_prophet#1355] [Q&A - get help using NeuralProphet]
I noticed the point forecasts in the test set shifted upwards and away from the actuals after specifying quantiles as in NeuralProphet(quantiles = [0.025, 0.975]), because I wanted to generate conformal prediction intervals in the next step. This increased MAPE from 0.034545 to 0.041148. Why is that?

What should I do to make the point forecasts stay the same while adding valid prediction intervals?

**Comment by leoniewgnr** [comment]
Did you set the random seed? `set_random_seed(0)'

**Reply by tuttoaposto** [reply]
Yes, I did.
